### ISSUE TITLE
Attention: Misleading/missing application feedback in selection of enzymes from DB

### ISSUE THREAD

**Issue by ju-rgen** [Open-Systems-Pharmacology/PK-Sim#1009]
When selecting enzymes from the DB (e.g. ABCB1) in the displayed list the bottom entry is "highlighted" by gray background color.
When selecting the top entry it is not clearly marked/highlighted except for a slightly visible frame.
After pressing next it is not visible, which enzyme is shown.
This behaviour could lead to mistakes in the selection. (In particular, if the same name is used for two entries)
![grafik](https://user-images.githubusercontent.com/25579536/48831347-f0ff7f00-ed76-11e8-8777-bf3485f0bb6d.png)
![grafik](https://user-images.githubusercontent.com/25579536/48831367-fceb4100-ed76-11e8-9c25-065eccf436ab.png)
![grafik](https://user-images.githubusercontent.com/25579536/48831384-05437c00-ed77-11e8-99a4-5e7d1fbdc089.png)
![grafik](https://user-images.githubusercontent.com/25579536/48831393-08d70300-ed77-11e8-81bc-0fd7ec3c782a.png)

**Comment by ju-rgen** [comment]
See TC 77.3

**Comment by msevestre** [comment]
I'd say that there is nothing to change here. In general, the view is bigger as we assume that people are actually going to look at what's going on. Please reopen if you think we need to do anything

**Comment by PavelBal** [comment]
I agree with @ju-rgen , the protein selection UI is misleading (imo).
One issue is the poor highlighting of the selected entry.

Another issue is the generally unintuitive list of search results. E.g., in the case of "ABCB1", there are two entries with the same gene name (the first two). However, they represent two **different** genes - ABCB1 and ABCB4! The "Name Type" column is not clear to me.

Also it is misleading, as the tab is called "Protein Selection", but the user actually selects the gene (makes sense as it es a gene expression database). Also, it would be good to also show a column with all other names that lead to this entry. E.g., in case of ABCB1, the same gene can be found by searching for "p-gp".

![image](https://user-images.githubusercontent.com/25903040/49221196-b205a500-f3d8-11e8-9e6c-e3e5315a8c25.png)

However, these are the same entires. Also, "P-GP" is listed as "Gene name", which is wrong - the gene is still ABCB1.

**Comment by msevestre** [comment]
@pavelbal a lot of issues in one single entry! They will all get lost. Would you please open one issue for each points that you wrote? I think those are all valid points Indeed.

This issue here is about selection. Clearly the first view should have better highlighting. Let's keep the focus in this idea here. Going to reopen

**Comment by Yuri05** [comment]
@PavelBal s. also discussions in #966 and #946
